js/Track/library/File/BIGWIG.js

```diff
@@ -0,0 +1,5 @@
+Genoverse.Track.File.BIGWIG = Genoverse.Track.Graph.Bar.extend({
+  model  : Genoverse.Track.Model.File.BIGWIG,
+  name   : 'bigwig',
+  height : 100
+})
```

Comment:
* Please add ; to the end of the file
* Also add the line ```Genoverse.Track.File.BW = Genoverse.Track.File.BIGWIG;``` to allow drag/drop to work (the file ending is .bw rather than .bigwig)